Automattic/jetpack
# Contact Form: Update/contact form allow layout blocks

Fixes #25123

#### Changes proposed in this Pull Request:
* Updates the Form block to allow the following blocks as child blocks: `Columns`, `Row`, `Stack`, and `Group`.
* All the Form child blocks should still exist as standalone blocks. By adding one of the child blocks, a Form block should be added to wrap the child block, along with a submit button.
* If nested in any level inside a Form block, the child block should be added without any wrapper.

#### Other information:


#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* Create a Post/Page and add a Form block
* Select a variation and play with the block to confirm if the blocks described above are now available as child blocks
* Add some child blocks inside the layout blocks and check if they work as expected
* Outside the Form block, add some child blocks and check if they will be added wrapped by the Form block
